### ISSUE TITLE
Close button should be first in PDOM order/tab order in snapshots dialog

### ISSUE THREAD

**Issue by brettfiedler** [phetsims/photoelectric-effect#272]
In the snapshots dialogs, when they are opened, the Close button is the first thing that keyboard focus sits on. The next tab takes a user out of the browser.

Ideally, we want the zoom buttons to be the next object to tab to. I think this means moving the close butto nto the top of the tab order?

**Comment by marlitas** [comment]
This was a really simple option change. Done. Over to @brettfiedler to confirm (fix should be on main), and close if it's behaving as desired.

**Comment by brettfiedler** [comment]
Fix is on `features/photoelectric-effect/issues/272`, not `main`, but it works! Good to merge with main and close.

**Comment by marlitas** [comment]
Hmmm interesting. I had it merged in but maybe it didn't push fully. Well confirmed its there now so ready to close.
